fitbenchmarking/core/fitting_benchmarking.py

```diff
@@ -606,7 +606,7 @@ def perform_fit(controller, options, grabbed_output):
 
         accuracy_check = any(np.isnan(n) for n in accuracy) \
             if controller.problem.multifit else np.isnan(accuracy)
-        if np.isnan(runtime) or accuracy_check:
+        if np.isnan(mean_runtime) or accuracy_check:
```

Comment:
I think this is the only time we use `mean_runtime` so we can probably just check the runtime list for nan instead?
```suggestion
        if np.isnan(runtimes).any() or accuracy_check:
```

Reply:
I will make this suggested change. I will change one minor thing also. I will remove the `mean_runtime` variable from `perform_fit()`. It is no longer being used.

Reply:
I think that you are right. It probably needs a refactor. A name change would be helpful and calculating the minimum runtime across each of the iteration might be more informative.

**My issue at the moment is thinking through how this will get translated on to the frontend.** 

I think we should just put one value for `runtime` into the tables (just as we are doing). And this `runtime` can be the mean, median, truncated (trimmed) mean where we are handling the outliers or something entirely different. But we should just display one value. (We can justify that value later with some analysis).

All the relative calculations can be done based on that `runtime` value. Because if we define `min_runtime`  as the `min(runtimes)` and if that value is not shown on the frontend then the relative values might not make sense to the end user.  

These are real runtimes for the `Misra1d.dat`. For now, I think `mean_runtime` might not be a bad estimate for runtime.

```
runtimes  :  [0.008867208991432562, 0.006633415992837399, 0.006206958991242573, 0.005718000000342727, 0.005523750005522743]
runtimes  :  [0.02062587499676738, 0.017507749987998977, 0.016206542000873014, 0.015608957997756079, 0.015931290996377356]       
runtimes  :  [0.009807166003156453, 0.009678250004071742, 0.009295042007579468, 0.009243665990652516, 0.009474917009356432]    
runtimes  :  [0.017122624994954094, 0.012101541011361405, 0.011961124997469597, 0.010736625001300126, 0.010084249995998107]      
runtimes  :  [0.002930749993538484, 0.0014395000034710392, 0.0014268749946495518, 0.0014097090024733916, 0.0014177080010995269]
runtimes  :  [0.002621042003738694, 0.0019692499918164685, 0.0018949579971376806, 0.001894082990474999, 0.0018268340063514188]   
runtimes  :  [0.0019839589949697256, 0.0014409589930437505, 0.0014155410026432946, 0.0014021249953657389, 0.0013256249949336052]      
runtimes  :  [0.0031089159892871976, 0.002127208004822023, 0.0020715839928016067, 0.0020633339881896973, 0.0016899579932214692]       
runtimes  :  [0.0008704999927431345, 0.0005838330107508227, 0.0005634170083794743, 0.0005653330008499324, 0.0005600409931503236] 
runtimes  :  [0.0015235830069286749, 0.0010692910000216216, 0.0010547089914325625, 0.0010596250067465007, 0.0009845000022323802]  
runtimes  :  [0.002643083003931679, 0.0018827080057235435, 0.001691208002739586, 0.0016667919990140945, 0.0016790410008979961]
runtimes  :  [0.0026474170008441433, 0.0019747920014197007, 0.0019439169991528615, 0.0019664999999804422, 0.0018281670054420829]
runtimes  :  [0.006368957998347469, 0.004968666995409876, 0.004439875003299676, 0.004154624999500811, 0.0037402500020107254]
runtimes  :  [0.05352941701130476, 0.03581895800016355, 0.0335104580008192, 0.033345124989864416, 0.03344670799560845] 
runtimes  :  [0.018600166993564926, 0.013525040994863957, 0.011692165993736126, 0.010837249996257015, 0.009858915989752859]
runtimes  :  [0.01952112499566283, 0.014283583004726097, 0.013488249998772517, 0.011509290998219512, 0.010713916999520734]
runtimes  :  [0.0011455420026322827, 0.0006961670005694032, 0.0006739169912179932, 0.0006697500066366047, 0.0006637920014327392]
runtimes  :  [0.0026254169933963567, 0.0017072079936042428, 0.0016442089981865138, 0.0015972499968484044, 0.0015065830084495246]
runtimes  :  [0.0037901250034337863, 0.002173667002352886, 0.001979584005312063, 0.0018032910011243075, 0.0018130419921362773]
runtimes  :  [0.001889916995423846, 0.0013212920021032915, 0.0012964170018676668, 0.0012940000015078112, 0.0012857090041507035]
runtimes  :  [0.0007472080033039674, 0.0005054170032963157, 0.0004933330055791885, 0.0004932910087518394, 0.0004904999950667843]
runtimes  :  [0.0013482920039677992, 0.0008022090041777119, 0.000783125011366792, 0.0007734579994576052, 0.0007722500013187528]
runtimes  :  [0.001650541991693899, 0.0009022919985000044, 0.0008737919997656718, 0.0008621250017313287, 0.0008657080034026876]
runtimes  :  [0.0016258339892374352, 0.000993084002402611, 0.0009746249997988343, 0.0009568750101607293, 0.0009518330043647438]
```